packages/browser/src/lib/frontend.ts

```diff
@@ -1,6 +1,15 @@
-import { FrontendBase, Sdk, SdkInfo } from '@sentry/core';
+import { createAndBind, FrontendBase, SdkInfo } from '@sentry/core';
 import { BrowserBackend, BrowserOptions } from './backend';
 import { Raven } from './raven';
+export { addBreadcrumb, captureEvent, setUserContext } from '@sentry/core';
+export {
+  captureException,
+  captureMessage,
+  popScope,
+  pushScope,
+  setExtraContext,
+  setTagsContext,
+} from '@sentry/shim';
```

Comment:
You could move these re-exports out of the frontend file, as it doesn't really belong here. Same for `create`.